lib/srv/discovery/access_graph.go

```diff
@@ -44,14 +42,28 @@ const (
 	batchSize = 500
 )
 
-func (s *Server) reconcileAccessGraph(ctx context.Context, currentTAGResources *aws_sync.Resources, stream accessgraphv1alpha.AccessGraphService_AWSEventsStreamClient, features aws_sync.Features) {
+var (
+	// errNoTAGFetchers is returned when there are no TAG fetchers.
+	errNoTAGFetchers = errors.New("no TAG fetchers")
```

Comment:
nit: I'd rename it to access graph. `TAG,` depending on the context, can mean multiple things.